boards/recovery/src/main.rs

```diff
@@ -49,6 +49,7 @@ mod app {
         led_green: Pin<PB16, PushPullOutput>,
         led_red: Pin<PB17, PushPullOutput>,
         state_machine: StateMachine,
+        recovery_timer: hal::timer::TimerCounter2,
```

Comment:
Could make this cleaner by doing a use hal::timer::TimerCounter 2 for the import then just use TimerCounter2 here.